compiler/damlc/tests/BUILD.bazel

```diff
@@ -689,8 +689,8 @@ da_haskell_test(
         "//compiler/damlc/stable-packages",
         "//compiler/repl-service/server:repl_service_jar",
         "//daml-script/daml:daml-script.dar",
-        "//ledger/sandbox-on-x:app",
-    ],
+        "@canton//:lib",
+    ] + (["@sysctl_nix//:bin/sysctl"] if is_darwin else []),
```

Comment:
naive question, why do we need this  ?

Reply:
For FreePort, in order to get the dynamic port range, we need `sysctl` on MacOS.
This is the same for FreePort in scala :)